megamek/src/megamek/common/EjectedCrew.java

```diff
@@ -68,6 +77,29 @@ public EjectedCrew(Entity originalRide) {
         }
     }
     
+    /**
+     * Used to set up an ejected crew for large spacecraft per rules in SO p27
+     * Multiple entities will be set up, each with a different strength
+     * @param originalRide - the launching spacecraft
+     * @param escapedThisRound - The number of people that got out this round
+     */
+    public EjectedCrew(Aero originalRide, int escapedThisRound) {
+        super();
+        setCrew(new Crew(CrewType.CREW));
+        setChassis(SPACE_EJECT_NAME);
+        setModel(originalRide.getDisplayName());
+
+        // Generate the display name, then add the original ride's name.
+        StringBuffer newName = new StringBuffer(getDisplayName());
+        newName.append(" of ").append(originalRide.getDisplayName());
```

Comment:
Kinda pointless to have a stringbuffer here, since it's not in a loop or anything.

Reply:
Fixed